src/main/org/firebirdsql/jdbc/InternalTransactionCoordinator.java

```diff
@@ -175,7 +175,7 @@ void handleConnectionClose() throws SQLException {
         coordinator.handleConnectionClose();
     }
 
-    private void setCoordinator(AbstractTransactionCoordinator coordinator) throws SQLException {
+    public void setCoordinator(AbstractTransactionCoordinator coordinator) throws SQLException {
```

Comment:
Do you really need this? Looking at https://github.com/asfernandes/fbjava/blob/master/src/fbjava/src/main/java/org/firebirdsql/fbjava/impl/InternalFBConnection.java#L41 using `setAutoCommit(false)` would achieve this as well.